airflow/config_templates/config.yml

```diff
@@ -2671,6 +2671,13 @@ kubernetes_executor:
       type: string
       example: '{ "total": 3, "backoff_factor": 0.5 }'
       default: ""
+    detailed_executor_logs:
+      description: |
+        Flag to have more details in kubernetes executor pod logs
```

Comment:
Can this be more specific than “more details”? What details are they? Why/when would people not want those details?

Reply:
@uranusjr thanks for the review. All the logs present under kubernetes executor do not contain a lot of data needed to have the best debugging experience.

Example:
When trying to trace the lifecycle of a task in the kubernetes executor, you currently must search first for the name of the pod created by the task, then search for the pod name in the logs. This means you need to be pretty familiar with the structure of the scheduler logs in order to search effectively for the lifecycle of a task that had a problem.

Some log statements like Attempting to finish pod do have the annotations for the pod, which include dag name, task name, and run_id, but others do not. For instance, Event: podname-a2f2c1ac706 had an event of type DELETED has no such annotations.

Reply:
I will rename the flag to something more meaningful like: `traceable-executor-logs`. What do you think @uranusjr?
Or any other alternative thoughts?

Reply:
Sounds good to me

Reply:
What about `logs_task_metadata`? if you look to the added information, they describes the task and not the executor. WDYT?